pact3.0/01-coin-gov.repl

```diff
@@ -0,0 +1,71 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Example of using module governance capability
+
+
+; --------------------------------------------------------------------------
+; Install account schema module
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-data { "admin-keyset": ["product-dept" "dev-dept"] })
+(module acct-module ACCT-GOVERNANCE
+  @doc "account schema module"
+
+  ;;; module can only be upgraded if admin-keyset is satisfied.
+  ;;; i.e. If transaction signed by both "product-dept" and "dev-dept"
+  (defun enforce-acct-admin ()
+    (enforce-guard (read-keyset "admin-keyset"))
+  )
+
+  (defcap ACCT-GOVERNANCE ()
+    (enforce-acct-admin)
+  )
+
+  (defschema account
+    balance:integer
+    ks:keyset))
+(commit-tx)
+
+; --------------------------------------------------------------------------
+; Upgrades schema by triggering module's governance capability.
+; --------------------------------------------------------------------------
+(begin-tx)
+;;; simulate multi-signature of product and dev department
+(env-keys ["product-dept" "dev-dept"])
+
+(module acct-module ACCT-GOVERNANCE
+  @doc "account schema module with 'active' column"
+
+  (defcap ACCT-GOVERNANCE ()
+    (enforce-guard (read-keyset "admin-keyset")))
+
+  (defschema account
+    balance:integer
+    ks:keyset
+    active:bool ;;; new column
+  ))
+(commit-tx)
+
+; --------------------------------------------------------------------------
+; Install hello-coin module
+; --------------------------------------------------------------------------
+(begin-tx)
+
+(module hello-coin HELLO-GOVERNANCE
+  @doc "Implementaion of hello-coin"
+
+  (use acct-module)
+
+  (defcap HELLO-GOVERNANCE ()
```

Comment:
the point was to use the (enforce-acct-admin) here, so maybe keep it around in your upgrade?